### PR TITLE
MDBF 633: Docker compose file example for MySQL 5.7 migration

### PR DESCRIPTION
**Pull request by an3l**

### CODE REVIEW

File: examples/migration-5.7/compose-migrate-mysql.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  MYSQL_ROOT_PASSWORD: ${DB_ROOT_PASSWORD}
+  MYSQL_USER: ${DB_USER}
+  MYSQL_PASSWORD: ${DB_PASSWORD}
+
+services:
+  mariadb-from-mysql57:
+    environment:
+      <<: *common-variables
+      MARIADB_AUTO_UPGRADE: 1
+    container_name: ${MYSQL_MIGRATE_name}
```
Comment (grooverdan) [inline examples/migration-5.7/compose-migrate-mysql.yml:12]: where is `MYSQL_MIGRATE_name` from?

File: examples/migration-5.7/compose-migrate-mysql.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+version: "3"
+x-common-variables: &common-variables
+  MYSQL_ROOT_PASSWORD: ${DB_ROOT_PASSWORD}
+  MYSQL_USER: ${DB_USER}
+  MYSQL_PASSWORD: ${DB_PASSWORD}
+
+services:
+  mariadb-from-mysql57:
+    environment:
+      <<: *common-variables
```
Comment (grooverdan) [inline examples/migration-5.7/compose-migrate-mysql.yml:10]: common variables aren't used when there's an upgrade. Remove?

**Review by grooverdan — COMMENTED** [review]
feel free to merge after consideration

File: examples/migration-5.7/compose-migrate-mysql.yml (hunk 2 above)
Comment (an3l) [reply examples/migration-5.7/compose-migrate-mysql.yml:10]: Yes, I will .

File: examples/migration-5.7/compose-migrate-mysql.yml (hunk 1 above)
Comment (an3l) [reply examples/migration-5.7/compose-migrate-mysql.yml:12]: From .env file

Comment (cvicentiu) [reply examples/migration-5.7/compose-migrate-mysql.yml:9]: Why not MARIADB_ROOT_PASSWORD and the other alternative MARIADB_ env vars?

**Pull request merged**